ScnSoft-Education/LeetCodeTraining/FourSum16/FourSum.cs

```diff
@@ -0,0 +1,49 @@
+﻿namespace LeetCodeTraining.FourSum16
+{
+    public class Solution
+    {
+        public IList<IList<int>> FourSum(int[] nums, int target)
+        {
+            List<IList<int>> result = new List<IList<int>>();
+
+            Array.Sort(nums);
+
+            for (int firstNumIndex = 0; firstNumIndex < nums.Length - 3; firstNumIndex++)
+            {
+                if (firstNumIndex > 0 && nums[firstNumIndex] == nums[firstNumIndex - 1]) continue;
+                for (int secondNumIndex = firstNumIndex + 1; secondNumIndex < nums.Length - 2; secondNumIndex++)
+                {
+                    if (secondNumIndex > firstNumIndex + 1 && nums[secondNumIndex] == nums[secondNumIndex - 1]) continue;
```

Comment:
don't volatile your style code!
'continue' should go from the new line and in curly brackets

Reply:
code has been brought to a single style